tests/main/views/test_users.py

```diff
@@ -2107,23 +2107,54 @@ def test_published_service_count_with_different_statuses(self):
 
 
 class TestUsersEmailCheck(BaseUserTest):
-    def test_valid_email_is_ok_if_domain_found_in_database(self):
+    def _setup(self):
```

Comment:
A small thing but can we either give this method a more descriptive name (e.g. `def _setup_buyer_domain`) or make it a proper `def setup(self)` method that applies to all tests in the class?

Reply:
Good spot! Have done latter suggestion in https://github.com/alphagov/digitalmarketplace-api/pull/1037/commits/15603b993ea2aff5b90e5411fa767f0a8302f1c5